promisasu/Portal
# Request Logging

**Taiga User Story**
362

**Taiga Task(s)**
551, 552

**What did you change?**
* Added request and response logs for API and Dashboard
* Added CPU and memory logs
* Ensured views return correct response code on error pages
* Added timestamps to recorded `console.log` and `console.error` events

**How can we test?**
* Run `npm install`
* Click around in Dashboard, use application to navigate through API
* `logs/prp-development-api-access.log` and `logs/prp-development-dashboard-access.log` should have a record of the requests
